DigitalLearningSolutions.Data/Services/LearningResourceReferenceService.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using DigitalLearningSolutions.Data.DataServices;
+
+    public interface ILearningResourceReferenceService
+    {
+        string? GetLearningHubResourceLinkByResourceRefId(int learningResourceReferenceId);
+    }
+
+    public class LearningResourceReferenceService : ILearningResourceReferenceService

```

Comment:
I don't think we'll need this if we're using LearningHubResourceService instead.